platformio.ini

```diff
@@ -624,3 +624,19 @@ build_flags = -c -H -std=gnu++11 -Wall -Os -D__MARLIN_FIRMWARE__
 lib_deps    = ${common.lib_deps}
   TMC26XStepper=https://github.com/trinamic/TMC26XStepper/archive/master.zip
 src_filter  = +<src/Marlin.cpp>
+
+#
+# Chitu boards like Troxy X5s (STM32F103ZET6)
+#
+[env:chitu_f103]
+platform      = ststm32
+framework     = arduino
+board         = genericSTM32F103ZE
+extra_scripts = buildroot/share/PlatformIO/scripts/chitu_crypt.py
+
```

Comment:
extra line to remove here, else could be a good practice to place it near the other F1 boards... before mks robin

Reply:
Fixed.

Reply:
nice to see a real/proper use of that... instead of an useless encryption without crc ;)